lib/picker.date.js

```diff
@@ -1222,16 +1222,15 @@ DatePicker.prototype.nodes = function( isOpen ) {
 
                                             return klasses.join( ' ' )
                                         })([ settings.klass.day ]),
-                                        'data-pick=' + targetDate.pick + ' ' + _.ariaAttr({
+                                        'data-pick=' + targetDate.pick + ' ' + 'id=' + targetDate.pick + ' tabindex="0" ' + _.ariaAttr({
```

Comment:
adding id to element to support activedescendent
making each date keyboard accessible

Reply:
Not sure if this works well with multiple datepickers (duplicate IDs). Do we have some other generated instance string that we can append to the id?